package Demo8;
import java.util.*;
public class Merge {
	public static Set<Integer> Merge(int[] a, int[] b) {
	    HashSet<Integer> set = new HashSet<>();
	    for(int num : a)
	    	set.add(num);
	    for(int num : b)
	    	set.add(num);
	    return set;
}
		public static void main(String[] args) {
			int[] a = {1,2,3,4};
			int[] b = {4,5,6,7};
			System.out.println(Merge(a,b));
		}

}
